app/views/service/_svcs_show.html.haml

```diff
@@ -31,8 +31,6 @@
           .col-md-12.col-lg-6
             %h3
```

Comment:
@karelhala I think https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/views/service/_svcs_show.html.haml#L33-L34 should be moved to bottom as well and https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/views/service/_svcs_show.html.haml#L30-L31 should be removed.